formbricks/formbricks
# Moved /components/shared to /app/shared

## What does this PR do?

Moving components from `apps/web/components` to `apps/web/app` directory. This is done because from next.js 13 and the app directory you can put all the components next to the pages in the app directory to have everything at one place and show where these files are used.

Fixes # https://github.com/formbricks/formbricks/issues/527



## Type of change




## How should this be tested?


The changes was tested by running the formbricks in locally and to check for any change of appearances as we moving frontend components from one directory to another.

## Checklist
